src/pages/Notebook.pug

```diff
@@ -56,241 +50,188 @@ block article
+        We performed - again - a TdT tailing reaction with the NEB TdT and changed the AT-rich primer concentration to 20 nM, the incubation temperature to room temperature and 30 C° and the incubation times. After that, we isolated DNA from the produced gel to recover elongated ssDNA strands and measured their DNA concentration via the NanoDrop.  
+        Moreover, we repeated the TdT tailing reaction with the ThermoFisher TdT and also changed the AT-rich primer concentration to 20 nM, the incubation temperature to room temperature and 37 C° and the incubation times. Subsequently, we repeated the TdT tailing reaction with the ThermoFisher TdT again because we used the wrong buffer volume the day before.         
+        In another experiment, we investigated the TdT tailing reaction of samples that either contain or do not contain CoCl2 at room temperature and 37 °C, to see if the elongation occurred independently of the co-factor. Then, we prepared a dNTP dilution of 100 µM and performed a TdT tailing reaction with the ThermoFisher TdT to compare the impact of the usage of dATP, dTTP, dGTP and dCTP on the reaction. 
         
         # Week 25
 
-        To get a standard gel reference for the ThermoFisher TdT we performed a TdT reaction containing a single type of dNTP. 
-        We repeated this experiment because it did not work the first time.         
-        In another experiment, we tested different incubation times with dATP at room temperature and 37 °C to investigate the optimal TdT reaction conditions.         
-        We also repeated the TdT tailing reaction with the ThermoFisher TdT. 
-        This time the TdT reactions contained a single type of dNTP incubated at room temperature and 37 °C for 30 min to get a standard gel reference for the ThermoFisher TdT.        
-        Then we performed a time-dependent TdT reaction, where we tested different incubation times to estimate the speed at which nucleotides were added.        
-        Additionally, a TdT tailing reaction with different buffer concentrations was performed to test the impact on elongation.        
-        We also performed a two-step TdT reaction, where we did a TdT tailing reaction, inactivated it, purified a part of the ssDNA and performed another TdT tailing reaction with the purified DNA to verify its impact on the elongation.       
-        Then we performed a TdT tailing reaction with different TdT concentrations and dGTP to investigate if it creates sharper bands. 
-        
-        Finally, we performed a vector digestion for cloning with a pMA_217 vector and the enzymes Kpnl and Sacl. 
+        To get a standard gel reference for the ThermoFisher TdT we performed a TdT reaction containing only a single type of dNTP. We repeated this experiment because it did not work the first time.         
```

Comment:
```suggestion
        To get a standard gel reference for the ThermoFisher TdT we performed a TdT reaction containing only a single type of dNTP. 
        We repeated this experiment because it did not work the first time.         
```